src/pages/_document.tsx

```diff
@@ -0,0 +1,56 @@
+import Document, {
+    Html,
+    Head,
+    Main,
+    NextScript,
+    DocumentContext,
+} from "next/document";
+import {
+    Components,
+    Env,
+    fetchDecoratorReact,
+    Props,
+} from "@navikt/nav-dekoratoren-moduler/ssr";
+import React from "react";
+import {getBreadcrumbs} from "../utils/innloggetStatus";
+import {getMiljø} from "../utils/miljøUtils";
+
+const decoratorEnv = "prod" as Exclude<Env, "localhost">;
+
+const decoratorParams: Props = {
+    env: decoratorEnv,
+    context: "arbeidsgiver",
+    breadcrumbs: getBreadcrumbs(getMiljø())
+};
+
+class _Document extends Document<{ decorator: Components }> {
+    static async getInitialProps(ctx: DocumentContext) {
```

Comment:
Hvorfor `getInitialProps` i stedet for `getServerSideProps`? Funker sikkert fint, jeg bare lurer